tests/test_energy_model.py

```diff
@@ -167,9 +167,9 @@ def test_single_small(self):
                                                    [0, 1, 0, 0]])
 
     def test_single_big(self):
-        placements = em.get_optimal_placements({'task0': 350})
-        self.assertPlacementListEqual(placements, [[0, 0, 350, 0],
-                                                   [0, 0, 0, 350]])
+        placements = em.get_optimal_placements({'task0': 300})
```

Comment:
Now that the knob is exposed, it would be nice to test it here. So we could keep the old test with `capacity_margin_pct=0`, and then re-run that test with e.g. `capacity_margin_pct=20` and make sure that it raises an exception.

Reply:
Sounds good. I added `test_capacity_margin_single()`.